src/languages/lsl.js

```diff
@@ -64,7 +64,8 @@ function(hljs) {
                 variants: [
                     hljs.COMMENT('//', '$'),
                     hljs.COMMENT('/\\*', '\\*/')
-                ]
+                ],
```

Comment:
Is this really part of this PR? Although a related topic, it doesn't match the PR title

Reply:
No idea, hahaha.

Reply:
I had to tweak a few things to make the auto-detect happy.